src/masternodes/mn_checks.cpp

```diff
@@ -1352,26 +1352,54 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor
             return Res::Err("tx not from foundation member");
         }
         for(const auto& var : obj.govs) {
-            auto result = var->Validate(mnview);
-            if (!result) {
-                return Res::Err("%s: %s", var->GetName(), result.msg);
+            auto res = var->Validate(mnview);
+            if (!res) {
+                return Res::Err("%s: %s", var->GetName(), res.msg);
             }
-            // Make sure ORACLE_BLOCK_INTERVAL only updates at end of interval
+
             if (var->GetName() == "ORACLE_BLOCK_INTERVAL") {
+                // Make sure ORACLE_BLOCK_INTERVAL only updates at end of interval
                 const auto diff = height % mnview.GetIntervalBlock();
                 if (diff != 0) {
                     // Store as pending change
                     storeGovVars({var, height + mnview.GetIntervalBlock() - diff});
                     continue;
                 }
+            } else if (var->GetName() == "ATTRIBUTES") {
+                // Add to existing ATTRIBUTES instead of overwriting.
+                auto govVar = mnview.GetVariable(var->GetName());
```

Comment:
Check for nullptr, just preventing unexpected error.
It can happen if authorized person do a mistake, it should not shutdown entire network.

Reply:
nullptr checked for here.

https://github.com/DeFiCh/ain/blob/0b63d4f4bd5099ef6b24689debf2fd6568d75862/src/masternodes/mn_checks.cpp#L380

Reply:
Actually it's fine. I was a bit for `mnview.GetVariable(var->GetName())` to return `nullptr`, but it's actually checked and there is "ATTRIBUTES", in some weird situation if variable somehow does not exists it could be a problem. But probably it should not happen at all.